README.md

```diff
@@ -85,8 +77,8 @@ colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release --catkin-skip-building-test
 
 ### Set hardware configuration
 
-Prepare launch files and vehicle_description according to the sensor configuration of your hardware.  
-The following are the samples.
+Prepare launch and vehicle_description files according to the sensor configuration of your hardware.  
+The following files are provided as samples:

```

Comment:
This section could provide more details about how these files are prepared, how they are used, and where they should be stored in the workspace.

Reply:
There's more description about these files in the design docs (somewhere). I can update this to add a link to those explanations.